docs/documentation/glossary.css

```diff
@@ -0,0 +1,42 @@
+
+/* .styled-table tbody tr {
+    border-bottom: thin solid #dddddd;
+} */
+
+.styled-table th,
+.styled-table td {
+    border-color: rgba(0, 0, 0, 0);
+    padding: 20px 30px;
+
+}
+
+.styled-table th {
+    color:#fff;
+}
+
+.styled-table td {
+    color:#444;
+}
+
+.styled-table thead tr {
+    background-color: #5f3ece;
+    color: #ffffff;
+    text-align:center;
+    border-color: rgba(0, 0, 0, 0);
+
+}
+
+.styled-table {
+    border-collapse: collapse;
+    margin: 25px 0;
+    font-size: 0.9em;
+    /* font-family: Karla sans-serif; */
+    min-width: 400px;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0);
+    border-radius: 10px;
+}
+
+.glossaryTerm {
+    font-size: 20%;
+    font-weight: bold;
+}
```

Comment:
Are these being used at all? If not, please remove.

Reply:
Fixed in https://github.com/Arrow-air/website/pull/44/commits/23f391a6292061ac0c46f1a29b6e161e59b0ee66